app.js

```diff
@@ -0,0 +1,41 @@
+function generateRandomNumber() {
+  let  number= Math.floor((Math.random()*100))
+    return number
+  
+} 
+
+
+function getUserGuess(){
+    enterNumber =parseInt( prompt(`input the number`))
```

Comment:
Please use the variable keyword when you define a variable:
`let  enterNumber =parseInt( prompt("input the number"))`